mongoose_dashboard/app.js

```diff
@@ -0,0 +1,90 @@
+var express = require('express');
+var app = express();
+var bodyParser = require('body-parser');
+var path = require('path');
+var mongoose = require('mongoose');
+
+mongoose.connect('mongodb://localhost/mongo_dashboard');
+
+var MongooseSchema = new mongoose.Schema({
+	name: {type: String}
+}, {timestamps: true});
+
+mongoose.model('Mongoose', MongooseSchema);
+
+var Mongoose = mongoose.model('Mongoose');
```

Comment:
We can actually combine these two lines, 13 and 15. It will read like 'var Mongoose = mongoose.model('Mongoose', MongooseSchema);'